test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -906,32 +906,40 @@ class M(torch.nn.Module):
             def __init__(self, use_bias):
                 super().__init__()
                 self.linear = torch.nn.Linear(4, 4, use_bias)
+                self.linear2 = torch.nn.Linear(4, 4, use_bias)
 
             def forward(self, x):
-                return self.linear(x)
+                return self.linear2(self.linear(x))
 
         bias_list = [True, False]
-        for bias in bias_list:
-            mod = M(bias).eval()
-            v = torch.randn((2, 4))
+        for int8_mixed_bf16 in (
```

Comment:
separate tests might be cleaner

Reply:
Thanks for the suggestions and split the tests for fp32 and bf16.